frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -353,6 +352,11 @@
     //! request status
     public static final String REQUEST_STATUS = "request_status";
 
+    // friendly url dummy values
```

Comment:
Maybe don't call them "dummy", how about something more specific, like "placeholder" - but these are just minor details...

Reply:
I renamed the friendly url portlet constants and added a prefix.

I think we reach the goal if we generate the friendly urls for the detail/edit pages.
Currently the portlet action like update or abort or other stuff are missing but we can easily enhance the routes for these scenarios. In addition to this I will describe the procedure in wiki how to use the friendly url mapping and how to create new routes for the portlets.